src/components/Cards/Cards.tsx

```diff
@@ -1,21 +1,14 @@
 import { useState } from 'react';
-
 import { CardInterface } from '../../constants/settings';
 import Card from '../_commonComponents/Card/Card';
-import { getCards } from '../../constants/preparation';
 
-const cardList = getCards();
+interface CardsProps {
+	cardList: CardInterface[],
+	removedCards: Object,
```

Comment:
Try to avoid using `Object` type. This type is to wide. If you don't know the shape of object, just use `Record<string, unknown>`